aws/policy/security-services.yaml

```diff
@@ -4,83 +4,106 @@ Statement:
-          - arn:aws:iam::aws:policy/IAMReadOnlyAccess
-          - arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM
-          - arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceRole
-          - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+          - 'arn:aws:iam::aws:policy/IAMReadOnlyAccess'
+          - 'arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM'
+          - 'arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceRole'
+          - 'arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole'
+
   - Sid: AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees
     Effect: Allow
     Action:
       - iam:ListAccountAliases
       - iam:ListPolicies
       - iam:ListInstanceProfiles
       - iam:GetUser
-      - waf:CreateXssMatchSet
-      - waf:UpdateXssMatchSet
-      - waf:GetXssMatchSet
-      - waf:DeleteXssMatchSet
+      - waf:CreateByteMatchSet
       - waf:CreateGeoMatchSet
-      - waf:UpdateGeoMatchSet
-      - waf:GetGeoMatchSet
-      - waf:DeleteGeoMatchSet
-      - waf:CreateSqlInjectionMatchSet
-      - waf:UpdateSqlInjectionMatchSet
-      - waf:DeleteSqlInjectionMatchSet
-      - waf:GetSqlInjectionMatchSet
       - waf:CreateIPSet
-      - waf:UpdateIPSet
-      - waf:GetIPSet
-      - waf:DeleteIPSet
+      - waf:CreateRateBasedRule
+      - waf:CreateRegexMatchSet
+      - waf:CreateRegexPatternSet
+      - waf:CreateRule
+      - waf:CreateRuleGroup
       - waf:CreateSizeConstraintSet
-      - waf:GetSizeConstraintSet
-      - waf:UpdateSizeConstraintSet
-      - waf:DeleteSizeConstraintSet
-      - waf:CreateByteMatchSet
-      - waf:GetByteMatchSet
-      - waf:UpdateByteMatchSet
+      - waf:CreateSqlInjectionMatchSet
+      - waf:CreateWebACL
+      - waf:CreateXssMatchSet
       - waf:DeleteByteMatchSet
-      - waf:CreateRegexMatchSet
-      - waf:GetRegexMatchSet
-      - waf:UpdateRegexMatchSet
+      - waf:DeleteGeoMatchSet
+      - waf:DeleteIPSet
+      - waf:DeleteRateBasedRule
       - waf:DeleteRegexMatchSet
-      - waf:CreateRegexPatternSet
-      - waf:UpdateRegexPatternSet
-      - waf:GetRegexPatternSet
       - waf:DeleteRegexPatternSet
-      - waf:CreateWebACL
-      - waf:GetWebACL
-      - waf:UpdateWebACL
+      - waf:DeleteRule
+      - waf:DeleteRuleGroup
+      - waf:DeleteSizeConstraintSet
+      - waf:DeleteSqlInjectionMatchSet
       - waf:DeleteWebACL
-      - waf:List*
+      - waf:DeleteXssMatchSet
+      - waf:GetByteMatchSet
+      - waf:GetChangeToken
+      - waf:GetChangeTokenStatus
+      - waf:GetGeoMatchSet
+      - waf:GetIPSet
+      - waf:GetRateBasedRule
+      - waf:GetRateBasedRuleManagedKeys
+      - waf:GetRegexMatchSet
+      - waf:GetRegexPatternSet
       - waf:GetRule
-      - waf:CreateRule
+      - waf:GetRuleGroup
+      - waf:GetSizeConstraintSet
+      - waf:GetSqlInjectionMatchSet
+      - waf:GetWebACL
+      - waf:GetXssMatchSet
+      - waf:ListActivatedRulesInRuleGroup
```

Comment:
The only waf Action this list doesn't include is `ListLoggingConfigurations`. In light of the ongoing struggle with 6144 character policy limits we need to look at making more things use wildcards for low-risk actions, like List and Describe.  I'd prefer to keep the List* here.